src/dotty/tools/dotc/core/Types.scala

```diff
@@ -2527,6 +2527,8 @@ object Types {
       case _: MethodType => true
       case _ => false
     }
+    
+    def isTypeLambda: Boolean = variances.nonEmpty
```

Comment:
It would be nice to have a comment here, since it's not clear exactly why a `PolyType` is a type lambda if the variances are non-empty. Is it because a `PolyType` without parameters is not a type lambda?